### PR TITLE
[Behat] Order taxes aggregation scenarios

### PR DESCRIPTION
**Pull request by Zales0123**
| Q               | A
| --------------- | ---
| Bug fix?        | no
| New feature?    | no
| BC breaks?      | no
| Related tickets |
| License         | MIT

### CODE REVIEW

File: features/order/managing_orders/seeing_order_aggregated_taxes.feature [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    Background:
+        Given the store operates on a single channel in "France"
+        And there is a zone "EU" containing all members of the European Union
+        And default tax zone is "EU"
+        And the store has "EU VAT" tax rate of 23% for "Quantum mechanics" within "EU" zone
+        And the store has "EU Low VAT" tax rate of 10% for "Theoretical physics" within "EU" zone
+        And the store has a product "Schrödinger's cat" priced at "€100.00"
+        And it belongs to "Quantum mechanics" tax category
+        And the store has a product "General relativity" priced at "€50.00"
+        And it belongs to "Theoretical physics" tax category
```
Comment (pjedrzejewski) [inline features/order/managing_orders/seeing_order_aggregated_taxes.feature]: I enjoy these "Easter Eggs" too, but they make sense for irrelevant info, like customer name. Here we should use something that will be easy to understand and as close as possible to a real world scenario.

Comment (Zales0123) [inline features/order/managing_orders/seeing_order_aggregated_taxes.feature]: 🆗 😞

File: features/order/managing_orders/seeing_order_aggregated_taxes.feature [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+        And the store has "EU VAT" tax rate of 23% for "Bows" within "EU" zone
+        And the store has "EU Low VAT" tax rate of 10% for "Swords" within "EU" zone
+        And the store has a product "Composite bow" priced at "€100.00"
+        And it belongs to "Bows" tax category
+        And the store has a product "Claymore" priced at "€50.00"
+        And it belongs to "Swords" tax category
+        And the store has a product "Bastard sword" priced at "€150.00"
+        And it belongs to "Swords" tax category
+        And the store has "DHL" shipping method with "€10.00" fee within "EU" zone
+        And shipping method "DHL" belongs to "Bows" tax category
```
Comment (michalmarcinkowski) [inline features/order/managing_orders/seeing_order_aggregated_taxes.feature]: `"DHL" belongs to "Bows" tax category` - I would say it does not fit 😄

**Comment by pjedrzejewski** [comment]
Thank you Mateusz!

**Pull request merged**